examples/sanity/app/root.jsx

```diff
@@ -38,6 +38,7 @@ function Document({ children, title }) {
 }
 
 export default function App() {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
If this is unused we should just remove it.

Reply:
I thought it was used for some sanity test or so, that's why I disabled the rule here